eng/CdacPackageItems.props

```diff
@@ -2,21 +2,20 @@
   <!--
     *MicrosoftDotNetCdacTransportVersion is from Versions.props. This is how we integrate the cDAC transport packs into our builds
   -->
+  <PropertyGroup>
+    <_cdacPackageVersion Condition="'$(TargetRid)' == 'win-x64'">$(runtimewinx64MicrosoftDotNetCdacTransportVersion)</_cdacPackageVersion>
```

Comment:
silly nit - do we ever foresee them being versioned separately? If not - pick one. It'd be best if we had the IL only one, but we don't yet so let's leave this for cleanup.